.gitmodules

```diff
@@ -0,0 +1,8 @@
+[submodule "Sources/protobuf/protobuf"]
+	path = Sources/protobuf/protobuf
+	url = https://github.com/protocolbuffers/protobuf
+	branch = main
```

Comment:
While I like tracking form a breakage pov, but we likely need releases to be pinned or a release could break when it gets a new protobuf that needs changes to still build.

Maybe the right process is document how/when to update both of these but then also create a weekly cron CI step that updates both branches and tries building just to track when main changes in a way that would be a problem.  Just so we get early warning when something will need to be done.